docs/README.md

```diff
@@ -0,0 +1,351 @@
+# Test and config files
+
+A typical `*-bench` project runs checks specified in `controls` files that are a YAML 
+representation of the CIS Benchmark checks. There is a 
+`controls` file per version and node type.
```

Comment:
The second sentence is Kubernetes-specific (I don't think all benchmarks have node type)